docs/quickstart.md

```diff
@@ -0,0 +1,760 @@
+It is then automatically downloaded by SAM and added to lambda zip file. 
+
+!!! tip
+    If you are interested in learning how python decorators work under the hood you can follow great article from [Real Python](https://realpython.com/primer-on-python-decorators/).
+## Structured Logging
+In next step you have been tasked to propose production grade logging capabilities to your lambda code.
+You decided to follow [structured logging approach](https://docs.aws.amazon.com/lambda/latest/operatorguide/parse-logs.html) which means that you don’t write hard-to-parse and hard-to-keep-consistent prose in your logs but that you log events that happen in a context instead. 
+We want our log event to be in JSON format. This will allow us to search through it easily in CloudWatch Logs or Cloudwatch Insight.
+Initial idea might be to initiate logger from standard library and use `pythonjsonlogger` library to simplify structured log creation.
+
+=== "app.py"
+
+    ```python hl_lines="1 4 8-13 20 26 31"
+    import logging
+    import os
+
+    from pythonjsonlogger import jsonlogger
+
+    from aws_lambda_powertools.event_handler.api_gateway import ApiGatewayResolver
+
+    logger = logging.getLogger("APP")
+    logHandler = logging.StreamHandler()
+    formatter = jsonlogger.JsonFormatter(fmt="%(asctime)s %(levelname)s %(name)s %(message)s")
+    logHandler.setFormatter(formatter)
+    logger.addHandler(logHandler)
+    logger.setLevel(os.getenv("LOG_LEVEL", "INFO"))
+
+    app = ApiGatewayResolver()
+
+
+    @app.get("/hello/<name>")
+    def hello_name(name):
+        logger.info(f"Request from {name} received")
+        return {"statusCode": 200, "body": {"message": f"hello {name}!"}}
+
+
+    @app.get("/hello")
+    def hello():
+        logger.info(f"Request from unknown received")
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+    def lambda_handler(event, context):
+        logger.debug(event)
+        return app.resolve(event, context)
+    ```
+=== "requirements.txt"
+
+    ```bash
+    aws-lambda-powertools
+    python-json-logger
+    ```
+So in our first attempt we create new APP Logger. We configure Log Handler, formatter and log level. Then we use this logger to log required information. We will see logs structured like that
+```json
+{"asctime": "2021-11-22 15:32:02,145", "levelname": "INFO", "name": "APP", "message": "Request from unknown received"}
+```
+instead of
+```json
+[INFO]  2021-11-22T15:32:02.145Z        ba3bea3d-fe3a-45db-a2ce-72e813d55b91    Request from unknown received
+```
+
+So far so good! You might also decide to add additional context into your logs.
+You can take it from lambda context or event. Downside of this approach is that we would add additional complexity to ensure that you always add those specific attributes in every place.
+Also, we added boilerplate code and even with additional third party library it is not straightforward. 
```

Comment:
Hypothesis without example, maybe leave it out and focus that powertools make logging easier.

Reply:
Removed that part and focused on benefit of using powertools:
`

So far, so good! To make things easier, we want to add extra context to the logs. We can extract it from lambda context or event. We have to make sure that we always add those specific attributes wherever a logger is used. Can we ensure that the required attributes are added automatically on our behalf without having to move them around? Yes! Powertools Logger to the rescue :-)
`